sleepylyze/nrem.py

```diff
@@ -72,6 +72,10 @@ def __init__(self, fname=None, fpath=None, match=None, in_num=None, epoched=Fals
             # load the data
             self.load_segment()
 
+            #remove channels
+            exclude = ['EOG_L','EOG_R', 'EKG']
+            self.metadata['channels'] = [x[0] for x in self.data.columns if x[0] not in exclude]
+
```

Comment:
a soft git reset is a good alternative to new commits that undo changes from previous commits